docs/Guides/Live Screen Reloading.md

```diff
@@ -0,0 +1,53 @@
+You can now enable live screen reloading in ProMotion!
+
+When you run `pm_live_screens` in the REPL, ProMotion will watch your `app/screens` folder for changes and then intelligently reload screens for you. It'll even detect when a superclass is reloaded!
+
+But alas, it's not entirely automatic. Here's a basic guide on how to use this powerful feature effectively.
+
+## Set up screens to reload
+
+Since it's impossible to know how every screen should be instantiated in every case, we're not able to completely reload the screen for you. Instead, we provide a hook for you to do teardown and rebuild.
+
+```ruby
+class MyScreen < PM::Screen
+  def on_load
+    my_view = UIView.new
+    my_view.backgroundColor = UIColor.grayColor
+    my_view.frame = [[ 100, 100 ], [ 100, 50 ]]
+    self.view.addSubview my_view
+    load_data
+  end
+
+  def load_data
+    @some_live_data = { jamon: "Holmgren" }
+  end
+
+  def on_live_reload
+    # teardown all views and data
+    self.view.subviews.each(&:removeFromSuperview)
```

Comment:
I think you're right, @squidpunch. Do the subviews remove and then the `on_load`.